plugins/funind/functional_principles_proofs.ml

```diff
@@ -1098,10 +1098,12 @@ let prove_princ_for_struct (evd:Evd.evar_map ref) interactive_proof fun_num fnam
     let get_body const =
       match Global.body_of_constant const with
 	| Some (body, _) ->
+          let env = Global.env () in
+          let sigma = Evd.from_env env in
 	     Tacred.cbv_norm_flags
 	       (CClosure.RedFlags.mkflags [CClosure.RedFlags.fZETA])
-	       (Global.env ())
-	       (Evd.empty)
+               env
+               sigma
```

Comment:
@forestjulien I believe you were commenting this line.

> I will need more tests

You mean manual tests?

Reply:
yes

Reply:
IMHO it would be better to extend the funind test suite, rather than rely on manual testing.

Reply:
Ok I'm convinced. The word "test" was probably erroneous. I wanted to convince myself we cannot have any problem in the future nor with an unknown contrib.